shortest_superseq_problem/shortest_superseq_test.cpp

```diff
@@ -0,0 +1,35 @@
+#include "shortest_superseq.h"
+
+#include <gtest/gtest.h>
+
+TEST(ShortestSuperseq, QuestionExample) {
+  std::vector<int> in1 = {1, 5, 9};
+  std::vector<int> in2 = {7, 5, 9, 0, 2, 1, 3, 5, 7, 9, 1, 1, 5, 8, 8, 9, 7};
+
+  SubArray test_result = ShortestSuperseq(in1, in2);
+
+  EXPECT_EQ(test_result.start, 7);
```

Comment:
with a tiny bit extra syntax sugar you may be able to write this: 
   EXPECT_EQ(ShortestSuperseq(in1, in2), {7, 10});
this may sound like a small improvement, but more compact test code typically translates to more and better tests written.